src/org/pahappa/systems/services/IncidentServiceImpl.java

```diff
@@ -41,7 +41,14 @@ public Incident saveIncident(Incident incident) throws Exception {
 
 	@Override
 	public Incident updateIncident(Incident incident) throws Exception {
-		// TODO Auto-generated method stub
+		for(Incident item: incidents){
+			if(item.getId() == incident.getId()){
+				item.setTitle(incident.getTitle());
+				item.setType(incident.getType());
+				item.setComment(incident.getComment());
+				return item;
```

Comment:
I think If the incident was found, you're supposed to throw an Exception, that the Update failed.

Reply:
Totally skipped my mind, thanx